pipeline/outputs/loki.md

```diff
@@ -26,6 +26,7 @@ Be aware there is a separate Golang output plugin provided by [Grafana](https://
 | line\_format | Format to use when flattening the record to a log line. Valid values are `json` or `key_value`. If set to `json`,  the log line sent to Loki will be the Fluent Bit record dumped as JSON. If set to `key_value`, the log line will be each item in the record concatenated together \(separated by a single space\) in the format. | json |
 | auto\_kubernetes\_labels | If set to true, it will add all Kubernetes labels to the Stream labels | off |
 | tenant\_id\_key | Specify the name of the key from the original record that contains the Tenant ID. The value of the key is set as `X-Scope-OrgID` of HTTP header. It is useful to set Tenant ID dynamically. ||
+| compress | Set payload compression in network transfer. Option available is 'gzip'||
```

Comment:
Let's follow the standard comment in others, e.g. Opensearch output: https://docs.fluentbit.io/manual/pipeline/outputs/opensearch#configuration-parameters

```suggestion
| compress | Set payload compression mechanism. The only available option is gzip. Default = "", which means no compression. ||
```

Reply:
Assuming the default is correct

Reply:
Thank you for the comment. That sounds reasonable.